tensorflow_addons/optimizers/lazy_adam.py

```diff
@@ -149,3 +149,6 @@ def _resource_scatter_operate(self, resource, indices, update, resource_scatter_
         }
 
         return resource_scatter_op(**resource_update_kwargs)
+
+    def get_config(self):
```

Comment:
Any reason this is needed?

Reply:
Yes, with keras 3 it throws an error that `LazyAdam` doesn't have a `get_config` method.